packages/react-mutation-mapper/src/component/column/Signal.tsx

```diff
@@ -64,29 +103,45 @@ export function signalSortMethod(
 export function getSortValue(
     signalData: IExtendedSignalMutation
 ): number | null {
-    return signalData.germlineFrequency || null;
+    if (signalData) {
+        return signalData.germlineFrequency || null;
+    }
+    return null;
```

Comment:
Not a big deal but maybe do this instead:

```suggestion
    return signalData ? signalData.germlineFrequency || null: null;
```